.github/workflows/deploy-to-staging.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy to staging
```

Comment:
rename the file to match other ordinal actions: `2-deploy-to-staging.yml`?

Reply:
Done!